lib/prime_by_strategy.rb

```diff
@@ -1,8 +1,8 @@
-require 'strategies'
+require_relative 'strategies'
 
 class PrimeByStrategy
-  def is_prime?(input, strategies = Strategies::PRIME_STRATEGIES.map(&:new))
-    return false if strategies.any?{|s| s.not_prime?(input)}
-    return true
+  def is_prime?(input)
+    !Strategies::PRIME_STRATEGIES.map { |strategy| strategy.new(input) }
+    .any?(&:not_prime?)
```

Comment:
return false if any strategies are not prime `==` return true if all strategies are prime.

[Commit to demo](https://github.com/johnnymo87/prime_patterns/commit/de6d24976d5b686e4e09760c779134c5ffa245d0)